src/js/components/Button/Button.js

```diff
@@ -97,7 +110,12 @@ const Button = forwardRef(
     let contents;
     if (first && second) {
       contents = (
-        <Box direction="row" align="center" justify="center" gap={gap}>
+        <Box
+          direction="row"
+          align="center"
+          justify={normalized ? 'start' : 'center'}
```

Comment:
I'm not sure why `justify` should be impacted by the `normalized` prop. what am I missing?

Reply:
Otherwise, the icon + label would be centered -- good call that maybe it shouldn't be forced. What do you think about giving users control of the justify? 

I think in some designs we have seen (such as sidebar), you would want the icon and label justified to the start as opposed to center.

Reply:
Interesting. can you share the designs? it will help with the context.

Reply:
Icon only:
<img width="80" alt="Screen Shot 2020-03-18 at 9 35 13 AM" src="https://user-images.githubusercontent.com/12522275/76984269-d4883e00-68fb-11ea-89f5-31b956327a73.png">

Icon + Label with justify="start":
<img width="297" alt="Screen Shot 2020-03-18 at 9 35 24 AM" src="https://user-images.githubusercontent.com/12522275/76984304-deaa3c80-68fb-11ea-8bb2-1fe4f3782a05.png">

Reply:
What about adding `align` to Button, taking values of `start`, `center`, `end`? I don't think the caller should care that Button has a Box in it, and I think `align` will seem a more natural term than `justify`.

Reply:
I agree `align` feels more natural, but it feels like it contradicts the language/behavior established by `align` vs `justify` in that in row/horizontal things, `justify` controls the horizontal positioning.

Reply:
I think we are more inclined to think that way because we know we have implemented this with a Box. I'm less sure that callers will think that justify fits better. But, I don't have a strong opinion on this. And, I don't think we have a strong precedent elsewhere to align with. Text uses `textAlign`, which does use the `align` term but with the `text` prefix, is different enough. So, I'm OK with `justify`. I think we should add that as a property for Button. I'm not sure that `normalized` implies `justify="start"` though. I'm wondering if `justify="center"` should be the default. Thoughts?

Reply:
Agreed that normalized shouldn't be the prop that drives it. I see what you're saying about justify and textAlign etc.. I would be flexible for the prop to be something other than justify if there are strong opinions in another direction. For now since you are OK with it, I will update Button to take justify, and agreed that center should be default because that is how Button currently works.